utils/download-singularity-img.sh

```diff
@@ -1,24 +1,40 @@
 #!/bin/bash
```

Comment:
So you actually have a nf script and a bash script to download singularity images?
One was not enough?

Reply:
This is because sometimes when singularity command is triggered it fails for some reason. So in case, people have this issue they can run the bash script instead. I might remove this in the future if people have no issues with this.